docs/index.rst

```diff
@@ -20,7 +20,9 @@ standalone or Kubernetes-based application gateway.
 
   intro/index
   intro/compatibility
-  user/QUICKSTART
```

Comment:
This should be careful, if you only change QUCkSTART.md to quickstart.md, no changes will be made to git workspace

Reply:
Sphinx will not find user/quickstart, so it will disappear..

Reply:
I thought I updated all QUICKSTART.md references. Am I missing any?

Reply:
Yes, you have updated all QUICKSTART.md referances, but the file name is still QUICKSTART.md not quickstart.md

Reply:
So this ` user/quickstart` will be not found.

Reply:
Or you should configure `git config core.ignorecase false` ?